### PR TITLE
MGDOBR-948: Add information about the transformation template language

### PR DESCRIPTION
**Pull request by vpellegrino**
**Task**: https://issues.redhat.com/browse/MGDOBR-948

image

### CODE REVIEW

**Review by sjamboro — APPROVED** [review]
LGTM

**Comment by vpellegrino** [comment]
I just need one review from either @kelvah or @manstis

File: src/app/Processor/ProcessorEdit/ProcessorEdit.tsx [hunk 1]
```diff
@@ -408,6 +411,37 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
...
+                                      />
+                                    }
+                                  >
+                                    <button
+                                      type="button"
+                                      aria-label={t(
+                                        "processor.moreInfoForTransformationTemplate"
+                                      )}
+                                      onClick={(e): void => e.preventDefault()}
+                                      aria-describedby="form-group-label-info"
```
Comment (kelvah) [inline src/app/Processor/ProcessorEdit/ProcessorEdit.tsx:439]: Minor obs.

You should probably update the value of `aria-describedby`. Maybe add an `id` attr to the parent `Text` component and use that?

**Review by kelvah — APPROVED** [review]
LGTM :ok_hand:

File: src/app/Processor/ProcessorEdit/ProcessorEdit.tsx (hunk 1 above)
Comment (vpellegrino) [reply src/app/Processor/ProcessorEdit/ProcessorEdit.tsx]: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/121/commits/3717fe2818801021fedb21855cdb3cc57762eac5

**Pull request merged**